src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+package src.main.java;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    static int totalNumberOfTask = 0;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        totalNumberOfTask++;
+    }
+
+    public String getStatusIcon() {
+        return (isDone? "\u2713": "\u2718");
+    }
+
+    public void completedTask() {
```

Comment:
Should this method name be of the form "setDone"/"setIsDone"? From the coding standard: "Setter methods for boolean variables must be of the form: `void setFound(boolean isFound);`